stable/restore/README.md

```diff
@@ -2,31 +2,15 @@
 
 This chart starts a Job to restore a NuoDB database from an existing backup, in a Kubernetes cluster using the Helm package manager.
```

Comment:
this document does not contain the "how to delete this deployment" section, that some others do

Reply:
It never did so I didn't add it. I wasn't sure if it is handled in another way, deleting a backup object, etc. deletes this. Sounds like you think it should be there so I'll add one. Done.